okio/src/jvmTest/kotlin/okio/BufferFactory.kt

```diff
@@ -59,7 +59,8 @@ enum class BufferFactory {
 
       return bufferWithRandomSegmentLayout(dice, largeByteArray)
     }
-  };
+  },
+  ;
```

Comment:
Arguably a ktlint bug

Reply:
I think this looks right. Trailing comma means clean diffs. Semicolon is required to separate constants from functions.

Reply:
I think this looks right. Trailing comma means clean diffs. Semicolon is required to separate constants from functions.